code/game/objects/items/devices/t_scanner.dm

```diff
@@ -65,72 +89,315 @@
 		user_client.images -= active_scanned[O]
 		active_scanned -= O
 
-	//Flicker effect
-	for(var/obj/O in active_scanned)
-		var/image/overlay = active_scanned[O]
-		if(flicker)
-			overlay.alpha = 0
-		else
-			overlay.alpha = 128
-	flicker = !flicker
 
-//creates a new overlay for a scanned object
+
+//creates a new overlay for a scanned object, if needed
 /obj/item/device/t_scanner/proc/get_overlay(obj/scanned)
 	//Use a cache so we don't create a whole bunch of new images just because someone's walking back and forth in a room.
 	//Also means that images are reused if multiple people are using t-rays to look at the same objects.
 	if(scanned in overlay_cache)
 		. = overlay_cache[scanned]
 	else
-		var/image/I = image(loc = scanned, icon = scanned.icon, icon_state = scanned.icon_state, layer = ABOVE_HUD_LAYER)
-		I.plane = ABOVE_HUD_PLANE
 
-		//Pipes are special
-		if(istype(scanned, /obj/machinery/atmospherics/pipe))
-			var/obj/machinery/atmospherics/pipe/P = scanned
-			I.color = P.pipe_color
-			I.overlays += P.overlays
-
-		I.alpha = 128
-		I.mouse_opacity = 0
```

Comment:
That line shouldn't be omitted. 
Without it holopipes will catch mouse click and prevent interactions with items on turf.

Reply:
its not working, tried adding it back but they still intercept clicks, see latest commit

I don't think its a big deal, pipes have safety checks so you still need to remove the floor to interact with them. You can turn off the scanner to do stuff, or put it in your pocket

Reply:
It's a big deal, because you can use it in combat to protect yourself. Mobs should be above those pipes anyway.

Reply:
Checked it.
That task is out of PR scope.